packages/http-client-csharp/generator/TestProjects/CadlRanch/http/type/union/src/Generated/EnumsOnly.cs

```diff
@@ -0,0 +1,34 @@
+// <auto-generated/>
+
+#nullable disable
+
+using System.ClientModel;
+using System.ClientModel.Primitives;
+using System.Threading.Tasks;
+using _Type.Union.Models;
+
+namespace _Type.Union
+{
+    public partial class EnumsOnly
+    {
+        protected EnumsOnly() => throw null;
+
+        public ClientPipeline Pipeline => throw null;
+
+        public virtual ClientResult Get(RequestOptions options) => throw null;
+
+        public virtual Task<ClientResult> GetAsync(RequestOptions options) => throw null;
+
+        public virtual ClientResult<GetResponse3> Get() => throw null;
```

Comment:
all implementation is `null`, so the test cases are not likely to run?

Reply:
they get regenerated before running the test this just avoids checking in many of the same generated code.